skops/card/_model_card.py

```diff
@@ -78,10 +78,10 @@ def metadata_from_config(config_path: Union[str, Path]) -> CardData:
 class Card:
     """Model card class that will be used to generate model card.
 
-    This class can be used to write information and plots to model card and
-    save it. This class by default generates an interactive plot of the model
-    and a table of hyperparameters. The slots to be filled are defined in the
-    markdown template.
+    This class can be used to write information and plots to model card and save
+    it. This class by default generates an interactive plot of the model and a
+    table of hyperparameters. The slots to be filled are defined in the markdown
+    template.
```

Comment:
hehe, seems like whatever tool you're using to format these, makes the lines one character longer than mine :D

Reply:
I used the plugin you sent though.

Reply:
hmm, somehow it's doing 80 for you and 79 for me.

Reply:
What plugin are you using?

Reply:
There seems to be a setting for the desired width, do you have the same one?

`"editor.wordWrapColumn"`

I installed the plugin and it seems to default to 80.

Reply:
I see, mine is set to 79.

Reply:
I have no preference.